src/main/plugins/org.dita.html5/xsl/simpletable.xsl

```diff
@@ -29,7 +29,14 @@ See the accompanying LICENSE file for applicable license.
     <xsl:variable name="col-widths" as="xs:double*">
       <xsl:variable name="widths" select="tokenize(normalize-space(@relcolwidth), '\s+')" as="xs:string*"/>
       <xsl:for-each select="$widths">
-        <xsl:sequence select="xs:double(substring(., 1, string-length(.) - 1))"/>
+        <xsl:choose>
```

Comment:
Updating to
```
<xsl:variable name="widths" select="tokenize(normalize-space(translate(@relcolwidth, '*', '')), '\s+')" as="xs:string*"/>
      <xsl:for-each select="$widths">
        <xsl:choose>
          <xsl:when test=". castable as xs:double">
            <xsl:sequence select="xs:double(.)"/>
          </xsl:when>
          <xsl:otherwise>
            <xsl:sequence select="xs:double(1)"/>
          </xsl:otherwise>
        </xsl:choose>
      </xsl:for-each>
```

Worth noting that this has a side effect of evaluating some invalid values --- by converting the `*` to an empty string, we end up treating all of these as columns with proportional widths of 1, 3 5:

- `relcolwidth="1* 3* 5*"`
- `relcolwidth="1 3 5"`
- `relcolwidth="*1 3* *5"` (this is the sort of typo that caused me to make this change initially)

I think that's probably fine - better than the current crash, and means we're doing what the author wanted even if the syntax is a bit off. But, just wanted to highlight it.